pkg/api/api.go

```diff
@@ -48,15 +48,23 @@ func New(profileCatalog Catalog) API {
 // ProfilesHandler is the handler for /profiles requests.
 func (a *API) ProfilesHandler(w http.ResponseWriter, r *http.Request) {
 	query := r.URL.Query().Get("name")
+	if query == "" {
```

Comment:
If you don't mind, would you please fix the comment here? https://github.com/weaveworks/profiles/pull/146/files#diff-67a6e36014be865053c43d38b2091bd2d3f17a91b117d97c0f5244cd7d71127dR20

:D No idea how it wasn't picked up by the linter. :/

Reply:
@Skarlso that link doesn't resolve to anything for some reason, what comment needs changing?

Reply:
eh, that's because github is stupido and it's a closed view.

https://github.com/weaveworks/profiles/blob/main/pkg/api/api.go#L20

This should work. :D

Reply:
done!